src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import model.FruitTransaction;
+import service.CsvFileReader;
+import service.CsvFileWriter;
+import service.FruitDao;
+import service.ReportGenerator;
+import service.ShopService;
+import service.impl.CsvFileReaderImpl;
+import service.impl.CsvFileWriterImpl;
+import service.impl.FruitDaoImpl;
+import service.impl.ReportGeneratorImpl;
+import service.impl.ShopServiceImpl;
+import strategy.OperationHandler;
+import strategy.OperationStrategy;
+import strategy.impl.BalanceOperation;
+import strategy.impl.OperationStrategyImpl;
+import strategy.impl.PurchaseOperation;
+import strategy.impl.ReturnOperation;
+import strategy.impl.SupplyOperation;
+
+public class Main {
+    private static final String inputPath = "src/main/resources/input.csv";

```

Comment:
read about constant naming convention
```suggestion
    private static final String INPUT_FILE = "src/main/resources/input.csv";
```